frame/support/src/lib.rs

```diff
@@ -1651,19 +1640,19 @@ pub mod pallet_prelude {
 /// 		Something(u32),
 /// 	}
 ///
-/// 	// Define a struct which implements `frame_support::traits::Get<T::Balance>`
+/// 	// Define a struct which implements `frame_support::traits::Get<T::Balance>`. (optional)
 /// 	#[pallet::type_value]
 /// 	pub(super) fn MyDefault<T: Config>() -> T::Balance { 3.into() }
 ///
-/// 	// Declare a storage, any amount of storage can be declared.
+/// 	// Declare a storage, any amount of storage can be declared. (optional)
```

Comment:
Same as above: should it be "a storage item"?
Same as above: periods go after the "(optional)".

Reply:
done through apopiak suggestion